### ISSUE TITLE
Integration with wit.ai

### ISSUE THREAD

**Issue by techscientist** [molybdenum-99/reality#53]
Hi, awesome project! I would love to see this grow to wolfram alpha's scale.

I think it would be very nice and useful to integrate this with wit.ai in order to understand natural language.

Great job!

**Comment by zverok** [comment]
Yeah, it looks like one of really cool future targets. Currently, I can't imagine where to start integrating it. 
Will be happy to see code ideas/pull requests, though!
